### DISCUSSION TITLE
Discussion group in gitter/slack/telegram

### DISCUSSION THREAD

**Discussion by jorgejesus** [ec-jrc/re3gistry#335] [Ideas]
# Background and Motivation

I am trying to understand how to create codelist on Re3gistry2, and followed the user docs but some basic aspects are not very clear specifically when you don't have experience with INSPIRE

# Proposed change

Have a gitter or slack channel where more users can interact and support each other with docs and howtos

# Alternatives

Can't find much on google (n00b), the following systems have helped me understand concepts and logic

- http://www.luettelopalvelu.fi/registry
- http://minerva.codes/registry

# Funding

No funding required

# Additional information

**Comment by df-git** [comment]
Thank you for your feedback, it is really useful.
We are trying to create a community, and we are assessing for the future the possibility to use one of the tools mentioned. For example we are currently submitting the Re3gistry to be included in the OSGeoLive distribution.
Would you please be more specific on the missing parts in the documentation?

**Comment by jorgejesus** [comment]
@df-git  Thank you for reaching out

Concerning docs:
- Docs about REST/API on re3gistry (for example is  it possible to use it input data?)
- For n00bs like me, a simpler guide (cookbook), starting with basic data and example so that the final output is something like this: http://minerva.codes/codelist/LandslideSizeClass/class10
- Open wiki or read the docs where a community could write content on how they worked with re3gistry v2 would be very useful
